docs/how_to_guides/how_to_use_apparent_and_true_chemical_species.rst

```diff
@@ -0,0 +1,213 @@
+How to use apparent and true chemical species
+=============================================
+
+In ProteusLib, most all chemical processes simulated will be considered "true"
+species, i.e., species that actually exist in an aqueous solution. However, their
```

Comment:
```suggestion
species, i.e., species that actually exist in an aqueous solution (e.g., Na+ and Cl- for NaCl). However, their
```
Already make sense and is clear, but I think it would be nice to spell this out as much as possible.